conda/recipes/kvikio/meta.yaml

```
@@ -44,23 +45,33 @@ requirements:
     - ninja
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} {{ cuda_version }}
+    {% else %}
+    - cuda-version {{ cuda_version }}
+    - {{ compiler('cuda') }}
+    {% endif %}
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
     - python
     - setuptools
     - pip
     - cython >=0.29,<0.30
-    - cudatoolkit {{ cuda_version }}.*
+    {% if cuda_major == "11" %}
+    - cudatoolkit ={{ cuda_version }}
+    {% endif %}
     - scikit-build >=0.13.1
-    - libkvikio {{ version }}
+    - libkvikio {{ version }}*=cuda{{ cuda_major }}*
```

Are we able to drop the build string here?

```suggestion
    - libkvikio {{ version }}
```

---

Yup using cuda-version now.